### PR TITLE
April 2019, nearly complete

### PR DESCRIPTION
**Pull request by jessedc**

### CODE REVIEW

File: intro-2019-04.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+---
+
+# Upcoming Events
+
+---
+
+# CocoaHeads Next
+
+- I am stepping down after the July event
+- Looking for help to share responsibilities and chape CocoaHeads going forward
```
Comment (cuteiosdev) [inline intro-2019-04.md]: s/chape/shape

File: intro-2019-04.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+- Drinks Night - April 30
+- Google Developers Group - April 24 
+
+---
+
+# Hack Night
+
+- Doors open at 6:00pm
+- Introductions from 6:30pm
+- BYO laptop and your iOS, macOS, tvOS, watchOS Android project
```
Comment (cuteiosdev) [inline intro-2019-04.md]: , after watchOS

File: intro-2019-04.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+- Google Developers Group - April 24 
+
+---
+
+# Hack Night
+
+- Doors open at 6:00pm
+- Introductions from 6:30pm
+- BYO laptop and your iOS, macOS, tvOS, watchOS, Android project
+- Alchohol free, :pizza: provided
```
Comment (tupps) [inline intro-2019-04.md]: Is 🍣 possible?

File: intro-2019-04.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+
+- Hack Night - April 17 @ Cognizant
+- Drinks Night - April 30
+- Google Developers Group - April 24 
+
+---
+
+# Hack Night
+
+- Doors open at 6:00pm
```
Comment (tupps) [inline intro-2019-04.md]: 5:45

File: intro-2019-04.md [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+- Introductions from 6:30pm
```
Comment (tupps) [inline intro-2019-04.md]: 6:15

File: intro-2019-04.md [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+## Follow up -> #jobs
+
+---
+
+# Presentations
+
+---
+
+# Next Hack Night :computer:
+## April 17 @ Cognizant from 6:00pm
```
Comment (tupps) [inline intro-2019-04.md:193]: 5:45

File: intro-2019-04.md [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+![inline 100%](assets/sponsors/YBF Ventures.png)
+
+---
+
+# Tonight's Agenda
+
+Time|Speaker|Topic|
+---|---|---
+6:40 - 6:55| Peter Goldsmith | The Month That Was (TMTW)
+7:00 - 7:25 | Kevin O'neal | iOS and TEA
```
Comment (kevinoneill) [inline intro-2019-04.md:175]: Kevin O'Neill

**Review by kevinoneill — CHANGES_REQUESTED** [review]

**Pull request merged**